templates/centos6.tpl.yaml

```diff
@@ -119,7 +119,7 @@ objects:
             interfaces:
             - masquerade: {}
               name: default
-              model: e1000e
+              model: virtio
```

Comment:
Shouldn't be this change in disk section? Because now you are updating network interface

Reply:
@ksimon1 I've updated the PR subject and description. The aim of this PR is to swtich both disk and network devices to work with virtio.